dex/src/main/scala/com/wavesplatform/dex/Matcher.scala

```diff
@@ -119,7 +120,7 @@ class Matcher(settings: MatcherSettings)(implicit val actorSystem: ActorSystem)
       assetPair,
       updateOrderBookCache(assetPair),
       marketStatuses.put(assetPair, _),
-      settings,
+      settings.orderFee match { case DynamicSettings(_, zeroMakerFee) => zeroMakerFee; case _ => false },
```

Comment:
Minor: probably move this logic to a method `MatcherSettings.zeroMakerFee`

Reply:
Done